Gherkin file for ID1

Alii279 opened the pull request:
Need a quick look on "Verifying Controls in the Browser emulator" and "Moving in VR" scenarios. I think we can delete them or modify them so please confirm.
I created scenarios manual test cases for:

1. Running the WebXR Vite Spike on the Meta Quest and on the browser emulator
2. Looking around and moving around the VR space/

randall-Github commented:
In terms of code coverage, do you think this covers enough for ID1?

Alii279 commented:
> In terms of code coverage, do you think this covers enough for ID1?

Yes, I believe I covered all the necessary issues and it would be sufficient for ID 1

randall-Github reviewed (approved):
Approved for merge

m-labrecque commented:
Why are there 2 tests that are the exact same except for the scenario? The scenarios are Verifying Controls in the Browser emulator and Running the Program on the browser emulator.
Why is there a test for using the Oculus Browser tabs? What is the purpose of that test, it has nothing to do with anything related to our project.
What is the purpose of the Scenario: Using the Right Controller in VR? To my knowledge those controls are not used in that way in VR. If you got those from the document about using the meta quest, that section was not for in the project it was for just using the browser on the meta quest.

Alii279 commented:
> Why are there 2 tests that are the exact same except for the scenario? The scenarios are Verifying Controls in the Browser emulator and Running the Program on the browser emulator. Why is there a test for using the Oculus Browser tabs? What is the purpose of that test, it has nothing to do with anything related to our project. What is the purpose of the Scenario: Using the Right Controller in VR? To my knowledge those controls are not used in that way in VR. If you got those from the document about using the meta quest, that section was not for in the project it was for just using the browser on the meta quest.

Yes, these two scenarios are duplicated but they weren't supposed to be that's my mistake. I have two different details for the two scenarios. I can redo it. 
For the right controller scenario, I was supposed to write tests based on the google doc that you told me to use to figure out the steps for my tests. I simply read the content of the document and thought that it is necessary to write this scenario. If you think that's unnecessary, feel free to remove it and commit anything you do in this branch.

m-labrecque commented:
I did not tell you to write tests based on the document about how to run the program, I told you to use the document for the steps to run the program on the Meta Quest. That test and the Oculus Browser test do not test our program they either test the meta quest itself or something is not implemented. Please remove these tests because they are not needed.
Please remove/rewrite the duplicated test and remove the tests not applicable to our project yourself this is your task to complete.

Alii279 commented:
> > Yes, these two scenarios are duplicated but they weren't supposed to be that's my mistake. I have two different details for the two scenarios. I can redo it. For the right controller scenario, I was supposed to write tests based on the google doc that you told me to use to figure out the steps for my tests. I simply read the content of the document and thought that it is necessary to write this scenario. If you think that's unnecessary, feel free to remove it and commit anything you do in this branch.
> 
> I did not tell you to write tests based on the document about how to run the program, I told you to use the document for the steps to run the program on the Meta Quest. That test and the Oculus Browser test do not test our program they either test the meta quest itself or something is not implemented. Please remove these tests because they are not needed. Please remove/rewrite the duplicated test and remove the tests not applicable to our project yourself this is your task to complete.

As mentioned above, I do feel like we require those tests as to cover all the basics and these are simple tests suitable for ID1. I have developed tests based on my understanding from our previous conversations and I gave you the permission to remove any tests that you don't require necessary. Nonetheless, I removed them now.
Deleted: Running the Program on the browser emulator/ Using the Oculus Browser Tabs / Using the Right Controller in VR
Please let me know if you need anything else.

m-labrecque reviewed (approved):
Looks good now, thank you for making those changes
